projects/core/src/store-finder/services/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './external-js-file-loader.service';
```

Comment:
folder `services` should be `service`. In `core`, all folder names are in singular.

Reply:
Folder name changed.